netbox/ipam/models/ip.py

```diff
@@ -125,11 +125,22 @@ class Meta:
         verbose_name_plural = 'ASNs'
 
     def __str__(self):
-        return f'AS{self.asn}'
+        if self.asn > 65535:
+            return 'AS{} ({}.{})'.format(self.asn, self.asn // 65536, self.asn % 65536)
+        else:
+            return f'AS{self.asn}'
 
     def get_absolute_url(self):
         return reverse('ipam:asn', args=[self.pk])
 
+    @property
+    def asdot_notation(self):
```

Comment:
Maybe rename this to `asn_with_asdot` or something like that, and return either the ASN with ASDOT notiation or just the ASN. Then you can reference it directly from within the ASNTable and object template.

Reply:
Great suggestion. That works much cleaner. Updated ASNTable & object template.